lib/atat-pipeline-stack.ts

```diff
@@ -46,12 +46,13 @@ class AtatApplication extends cdk.Stage {
         })
       );
     }
-    const monitoring = new AtatMonitoringStack(this, "AtatMonitoring", {
-      monitoredScopes: monitoredStacks,
-      notifiedEmail: props.notificationEmail,
-      environmentName: props.environmentName,
-      topicEncryptionKey: sharedData.encryptionKey,
-    });
+    // Removed the CDK monitoring stack
+    // const monitoring = new AtatMonitoringStack(this, "AtatMonitoring", {
+    //   monitoredScopes: monitoredStacks,
+    //   notifiedEmail: props.notificationEmail,
+    //   environmentName: props.environmentName,
+    //   topicEncryptionKey: sharedData.encryptionKey,
+    // });
```

Comment:
Let's remove this unless we plan to re-introduce in the near term.

Reply:
addressed/fixed